client/src/App.jsx

```diff
@@ -56,6 +56,16 @@ const App = () => {
 		getFetch();
 	}, []);
 
+	async function removeVideo(id){
+		console.log(`deleting video ${id}`);
+		const res = await fetch(`/api/videos/${id}`, {
+			method: "DELETE",
+		})
+		if(res.status === 204){
+			setVideos(videos.filter(video => video.id !== id));
```

Comment:
Calling getFetch function again then setting the videos according to updated data would be another option which would be more accurate. This way it is way faster and efficient on amount of data transferred. Good job!

Reply:
Thank you Fikret, I have changed the files according to the feedback but I am not sure about this.